src/aihwkit/nn/modules/linear.py

```diff
@@ -45,8 +46,6 @@ class AnalogLinear(AnalogModuleBase, Linear):
         weight_scaling_omega: the weight value where the max
             weight will be scaled to. If zero, no weight scaling will
             be performed.
-        digital_bias: decide whether the bias term is handled by the analog tile
```

Comment:
change decide to: determines whether the bias is handled by the analog tile or handled in digital domain.